oak_functions/loader/src/main.rs

```diff
@@ -48,6 +48,10 @@ struct Config {
     /// How often to refresh the lookup data. If not provided, data is only loaded once at startup.
     #[serde(with = "humantime_serde")]
     lookup_data_download_period: Option<Duration>,
+    /// Whether to use the GCP metadata service to obtain an authentication token for downloading
+    /// the lookup data.
+    #[serde(default)]
+    use_gcp_metadata_auth: bool,
```

Comment:
Can we configure one of the examples to use GCP authentication?

Reply:
The metadata service is only available to workloads running on GCP, so this won't work with local examples or on github CI.